virttest/vt_agent/managers/resource_backings/network/tap/tap_network_connection.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+
+from ...pool_connection import _ResourcePoolConnection
+
+
+LOG = logging.getLogger("avocado.agents.resource_backings.network.tap." + __name__)
+
+
+class _TapNetworkConnection(_ResourcePoolConnection):
```

Comment:
connect the guest network and entrance of host physical network card
3 checkpoints:
1. switch ( bridge )
2. up status
3. (optional) connected with physical network card